opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -172,6 +172,7 @@ where
         let mut log_record = self.logger.create_log_record();
         log_record.set_severity_number(severity_of_level(meta.level()));
         log_record.set_severity_text(meta.level().to_string().into());
+        log_record.set_target(meta.target().to_string());
```

Comment:
cijo - Can we use the static string instead of copy.

Reply:
We can't avoid the copy as the `metadata.target` is not the static str. It is str reference.